### PR TITLE
adds ability to set and guess multi-word phrases

### PR DESCRIPTION
**Pull request by NicoleCarpenter**

### CODE REVIEW

**Comment by zdavy** [comment]
:+1: on this PR, the only thing I want to note is that there seem to be a lot of `""`, `" "` and some other magic strings. I would suggest giving them some sort of semantic names, like `Tokens.EMPTY`, `Tokens.SPACE`, etc...

**Comment by zdavy** [comment]
In a different PR of course at the start of the next iteration

File: Sources/WordList/Words.swift [hunk 1]
```diff
@@ -1,22 +1,31 @@
...
+    "prayer", "elephant", "blink",
+    "doll", "spider", "point",
+    "balloon", "book", "glasses",
+    "stop", "sneeze", "mouth",
+    "draw", "football", "telephone",
+    "pig", "wave", "door",
+    "tail", "turtle", "baby",
+    "ear", "monkey", "hopscotch",
+    "mosquito", "toothbrush", "ring",
+    "hey jude", "twist and shout",
```
Comment (mnewsome) [inline Sources/WordList/Words.swift:22]: :dancers:

**Comment by mnewsome** [comment]
:+1:

**Pull request merged**